infra-templates/ecr/1/docker-compose.yml.tpl

```diff
@@ -5,10 +5,17 @@ ecr-updater:
     AWS_REGION: ${aws_region}
     AUTO_CREATE: ${auto_create}
     LOG_LEVEL: ${log_level}
+    {{- if eq .Values.registry_in_which_environment "other" }}
+    CATTLE_URL: ${environment_api_endpoint}
+    CATTLE_ACCESS_KEY: ${environment_api_access_key}
+    CATTLE_ACCESS_SECRET: ${environment_api_secret_key}
```

Comment:
@kingsd041 Did you test with `CATTLE_ACCESS_SECRET` or with `CATTLE_SECRET_KEY`? I believe this should be `CATTLE_SECRET_KEY`

Reply:
Agree, my mistake.